subsys/nrf_security/configs/psa_crypto_config.h.template

```diff
@@ -75,16 +75,21 @@
 #cmakedefine PSA_WANT_ECC_SECP_R1_384                           @PSA_WANT_ECC_SECP_R1_384@
 #cmakedefine PSA_WANT_ECC_SECP_R1_521                           @PSA_WANT_ECC_SECP_R1_521@
 
-#cmakedefine PSA_WANT_KEY_TYPE_DERIVE                           @PSA_WANT_KEY_TYPE_DERIVE@
-#cmakedefine PSA_WANT_KEY_TYPE_HMAC                             @PSA_WANT_KEY_TYPE_HMAC@
+/* The following key types are not configurable in the Oberon PSA implementation */
```

Comment:
How are we enforcing that we don't get a mismatch between KConfig and these C defines?

Reply:
offline discussion:
Decided to forward the Kconfig instead. Having these features accepted without the PSA_WANT symbol set to enabled is acceptable.